environment-approvals.yml

```
@@ -0,0 +1,3 @@
+prod:
+  - https://github.com/HMCTS/cnp-plum-recipes-service.git
```

I would remove hmcts and .git 👅 
should be easy to remove?
```suggestion
  - cnp-plum-recipes-service
```

---

i was thinking about future where we might be using something else e.g. azure DevOps

---

we can do the same there, and I don't think there would be any plan to move code to azure devops? we can't code in the open there...

---

would prefer to keep it explicit - it's a simpler comparison and avoids any potential bugs/edgecases in stripping this out or hard-coding prefixes and suffixes, etc